### PR TITLE
Fabo/802 artifactify logs

### LINKED ISSUE #802: Make app logs available after failed e2e tests.
UI Version: 0.6.2

Description:
Currently the log files are not accessible on the CI after e2e tests fail. It would be nice to have them as artifacts to allow better debugging.

### PR DESCRIPTION
**Pull request by faboweb**
Closes #802 

Description:

This now exports all the configs used for the test and all logs as an artifact.

❤️ Thank you!

### CODE REVIEW

File: test/e2e/launch.js [hunk 1]
```diff
@@ -156,22 +144,70 @@ async function printAppLog(app) {
...
   })
 }
 
+async function writeLogs(app, location) {
+  const mainProcessLogLocation = join(location, "main-process.log")
+  const rendererProcessLogLocation = join(location, "renderer-process.log")
+  fs.ensureFileSync(mainProcessLogLocation)
+  fs.ensureFileSync(rendererProcessLogLocation)
+
+  let mainProcessLogs = await app.client.getMainProcessLogs()
```
Comment (NodeGuy) [inline test/e2e/launch.js]: Please make this `const`.

File: test/e2e/launch.js [hunk 2]
```diff
@@ -156,22 +144,70 @@ async function printAppLog(app) {
...
+  let rendererProcessLogs = await app.client.getRenderProcessLogs()
```
Comment (NodeGuy) [inline test/e2e/launch.js]: Please make this `const`.

File: test/e2e/launch.js [hunk 3]
```diff
@@ -6,11 +6,12 @@ let electron = require("electron")
 let { join } = require("path")
 let { spawn } = require("child_process")
 let fs = require("fs-extra")
-let { newTempDir, login } = require("./common.js")
+let { login } = require("./common.js")
 
 const networkPath = join(__dirname, "localtestnet")
+const test_dir = join(__dirname, "../../test_temp")
```
Comment (NodeGuy) [inline test/e2e/launch.js]: `test_dir` should be in camelCase.

File: .circleci/config.yml [hunk 4]
```diff
@@ -37,6 +37,7 @@ jobs:
     environment:
       - BINARY_PATH: "/home/circleci/project/builds/gaia/linux_amd64/gaiacli"
       - NODE_BINARY_PATH: "/home/circleci/project/builds/gaia/linux_amd64/gaiad"
+      - CI: "true"
```
Comment (NodeGuy) [inline .circleci/config.yml]: This is already [set automatically by CircleCI](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables).

**Comment by NodeGuy** [comment]
What about this question?

> How about we rename test_temp to testArtifacts as a better description of what it's holding?

and getting rid of this:

https://github.com/cosmos/voyager/pull/803#pullrequestreview-126897490

**Comment by faboweb** [comment]
Done

**Pull request merged**